hack/make/tools.mk

```diff
@@ -0,0 +1,22 @@
+# List of tools.
+TOOLS = \
+    atlas \
+    ent \
+    go-enum \
+    goa \
+    golangci-lint \
+    gomajor \
+    gosec \
+    gotestsum \
+    migrate \
+    mockgen \
+    shfmt \
+    tparse \
+    workflowcheck
+
+# Main target to install all tools.
+tools: $(addprefix tool-, $(TOOLS))
```

Comment:
@sevein I'm curious about the "tool-" prefixing. Did you have namespace problems without the prefix?